src/Superpower/Model/TextSpan.cs

```diff
@@ -270,5 +270,53 @@ public bool EqualsValueIgnoreCase(string otherValue)
             }
             return true;
         }
+
+
```

Comment:
Nit - extra newline

Reply:
Fixed.

Reply:
> I realise now that you haven't made the changes you mention because these would require a polyfill to keep the netstandard2.0 target happy

Yeah.  As it is, if they're using .NET Standard 2.0, then they'd need to use the polyfill to use the range operator (`..`).

If they're using .NET Core (not sure what the minimum version is), it just works, without polyfill package.